src/api/todos.ts

```diff
@@ -0,0 +1,18 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodos = (userId: number, title: string) => {
+  return client.post<Todo>('/todos', {
+    title,
+    userId,
+    completed: false,
+  });
+};
```

Comment:
```suggestion
export const addTodos = (todo: Omit<Todo, 'id'>) => {
  return client.post('/todos', todo);
};
```

this is just example, you still create a `newTodo` variable in your `Header.tsx` file,  you can pass that object in that func